invenio_app_ils/records/views.py

```diff
@@ -9,28 +9,112 @@
 
 from __future__ import absolute_import, print_function
 
-from flask import Blueprint, abort, current_app, request
+from flask import Blueprint, abort, current_app, make_response, request
 from invenio_db import db
 from invenio_files_rest.models import ObjectVersion
 from invenio_files_rest.signals import file_downloaded
 from invenio_records_rest.utils import obj_or_import_string
 from invenio_records_rest.views import pass_record
 from invenio_rest import ContentNegotiatedMethodView
 from invenio_rest.errors import FieldError
+from webargs import fields
+from webargs.flaskparser import use_args
 
 from invenio_app_ils.errors import DocumentRequestError, StatsError
 from invenio_app_ils.permissions import need_permissions
 from invenio_app_ils.pidstore.pids import DOCUMENT_PID_TYPE, \
-    DOCUMENT_REQUEST_PID_TYPE, EITEM_PID_TYPE
+    DOCUMENT_REQUEST_PID_TYPE, EITEM_PID_TYPE, PATRON_PID_TYPE
 from invenio_app_ils.proxies import current_app_ils
+from invenio_app_ils.records.serializers.access import patron_list_serializer
 from invenio_app_ils.signals import record_viewed
 
 
+def create_access_list_blueprint(app):
+    """Create access list blueprint."""
+    blueprint = Blueprint("ils_access_list", __name__, url_prefix="")
+    endpoints = app.config.get("RECORDS_REST_ENDPOINTS", [])
+
+    # Get the list not the document
+    options = endpoints.get(PATRON_PID_TYPE, {})
```

Comment:
I don't think you should copy options for rest endpoint, e.g. csv serializer... You should have `application/json` only here